plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterCheckboxList.php

```diff
@@ -0,0 +1,156 @@
+<?php
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+/**
+ * Product Filter: Attribute Block.
+ */
+final class ProductFilterCheckboxList extends AbstractBlock {
+
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'product-filter-checkbox-list';
+
+
+	/**
+	 * Get the color value from the color attributes.
+	 *
+	 * @param string $key        The key of the color attribute.
+	 * @param array  $attributes The block attributes.
+	 * @return string
+	 */
+	private function get_color_attribute_value( $key, $attributes ) {
+		if ( $attributes[ $key ] ) {
+			return $attributes[ $key ];
```

Comment:
Should it be using `esc_attr` here?

```suggestion
			return esc_attr( $attributes[ $key ] );
```

Reply:
Good catch. I fixed this issue in 05b9d281b65fe854e9469ff1231fe4130a86122e. But instead of escaping here and in your another comment, I ended up escaping lately in the render method.